homework-002/major_cities.rb

```diff
@@ -0,0 +1,33 @@
+# ****************************************************************************
+#
+# Major Cities
+#
+# ****************************************************************************
+
+major_cities = {BC: ["Vancouver", "Victoria", "Prince George"], AB: ["Edmonton", "Calgary"]}
+
+def cities(obj)
+  obj.each do |x, y|
```

Comment:
try to use names that help the reader of your code (which will usually be you after enough time that you've forgotten how it works). I would use `...each do |province, cities|`. Then the `puts` statement becomes very easy to read.